crates/typst-library/src/visualize/gradient.rs

```diff
@@ -697,6 +697,7 @@ impl Gradient {
     }
 
     /// Returns the angle of this gradient.
+    /// Used only for Linear and Conic gradients, otherwise returns None.
```

Comment:
There should be a blank line after the first line so that it's a separate paragraph.